README.md

```diff
@@ -10,122 +10,108 @@ Android Auto for IDrive
-    - Google Assistant works the best, but Alexa, Bixby, and Cortana are also compatible
-    - The voice assistant app can be set as a hardware shortcut button for convenience
-  - Phone Notifications
-    - Popup about new notifications
-    - Supports Dismiss, Mark As Read, or other notification actions
-  - Google Maps
-    - Basic search and routing
-    - Includes some dark themes
-    - Not compiled by default, because showing Google Maps in a car is against the Maps API license
-  - Control of Android Auto compatible music apps
-    - Supports browsing and searching apps' music libraries
-    - Supports selecting from a list of currently-queued songs, as well as basic back/next control
-    - Integrates into the car's audio context, for automatic resume and hardware button control
-    - Supports controlling any active music session, even apps that aren't Android Auto compatible
-    - Automatically updates the screen to follow the active app
-    - Recommended compatible apps:
-      - Audiobooks and Podcasts:
-        - [Acast Podcast Player](https://play.google.com/store/apps/details?id=com.acast.nativeapp)
-        - [AntennaPod](https://play.google.com/store/apps/details?id=de.danoeh.antennapod)
-        - [The Bob & Tom Show](https://play.google.com/store/apps/details?id=com.radio.station.BOB.TOM)
-        - [Castbox](https://play.google.com/store/apps/details?id=fm.castbox.audiobook.radio.podcast)
-        - [Google Play Books](https://play.google.com/store/apps/details?id=com.google.android.apps.books)
-        - [iVooz](https://play.google.com/store/apps/details?id=com.ivoox.app)
-        - [Libro.fm Audiobooks](https://play.google.com/store/apps/details?id=fm.libro.librofm)
-        - [Listen Audiobook Player](https://play.google.com/store/apps/details?id=com.acmeandroid.listen)
-        - [Player FM](https://play.google.com/store/apps/details?id=fm.player)
-        - [Podcast Addict](https://play.google.com/store/apps/details?id=com.bambuna.podcastaddict)
-        - [Voice Audiobook Player](https://play.google.com/store/apps/details?id=de.ph1b.audiobook)
-      - Music Library
-        - [AIMP](https://play.google.com/store/apps/details?id=com.aimp.player)
-        - [Black Player](https://play.google.com/store/apps/details?id=com.musicplayer.blackplayerfree)
-        - [DSub](https://play.google.com/store/apps/details?id=github.daneren2005.dsub)
-        - [jetAudio HD](https://play.google.com/store/apps/details?id=com.jetappfactory.jetaudio)
-        - [Media Monkey](https://play.google.com/store/apps/details?id=com.ventismedia.android.mediamonkey)
-        - [Musicolet Music Player](https://play.google.com/store/apps/details?id=in.krosbits.musicolet)
-        - [Neutron Music Player](https://play.google.com/store/apps/details?id=com.neutroncode.mp)
-        - [Plex](https://play.google.com/store/apps/details?id=com.plexapp.android) (only music and podcasts)
```

Comment:
I think the ` (only music and podcasts)` should be integrated into hover text, it looks out of place among the new sea of icons.